test/parallel/test-buffer-tostring-binary.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const assert = require('assert');
```

Comment:
I did include this and also added `enoughTestMem` to be used in "mem-heavy" tests.